src/app.tsx

```diff
@@ -6,7 +6,6 @@ import { HashRouter as Router, Switch, Route, Redirect } from 'react-router-dom'
 import InfoPage from './pages/info'
 import WelcomePage from './pages/welcome'
 import RasaRoute from './routes/rasa'
-import QuestionnairePage from './pages/questionnaire'
```

Comment:
do we want to remove everything related to the `QuestionnairePage` as well?

Reply:
No yet, we'll leave it as is for now